projects/schematics/src/shared/constants.ts

```diff
@@ -35,7 +35,15 @@ export const CHECKOUT_SERVICE = 'CheckoutService';
 export const CHECKOUT_PAYMENT_SERVICE = 'CheckoutPaymentService';
 export const CHECKOUT_DELIVERY_SERVICE = 'CheckoutDeliveryService';
 export const CART_DATA_SERVICE = 'CartDataService';
+export const CART_SERVICE = 'CartService';
 export const ACTIVE_CART_SERVICE = 'ActiveCartService';
+export const ADDRESS_BOOK_COMPONENT_SERVICE = 'AddressBookComponentService';
+export const CHECKOUT_GUARD = 'CheckoutGuard';
+export const CHECKOUT_CONFIG = 'CheckoutConfig';
+export const CHECKOUT_CONFIG_SERVICE = 'CheckoutConfigService';
+export const EXPRESS_CHECKOUT_SERVICE = 'ExpressCheckoutService';
+export const ROUTER = 'Router';
```

Comment:
Can you move this constant at the top of class section? Other classes from dependencies are also declared there (eg. Store).